halide.cmake

```diff
@@ -305,7 +305,6 @@ endfunction()
 
 # Set the C++ options necessary for using libHalide.
 function(_halide_set_cxx_options TARGET)
-  set_target_properties("${TARGET}" PROPERTIES CXX_STANDARD 11 CXX_STANDARD_REQUIRED YES CXX_EXTENSIONS NO)
```

Comment:
I presume this is being removed to allow you to use something later than C++11... I suppose it's ok to just remove this and require the downstream consumer to configure at-least-C++11.

Reply:
Exactly, cmake function shouldn't overside C++ std options like that. Some people (like me) might prefer to use different version.